hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -322,7 +322,7 @@
         }
         { bits:   "7",
           name:   "flash_wp_l_0",
-          desc:   "1: not allowed; 1: allowed",
```

Comment:
This change does not belong to the PR. Could you factor this into a separate PR?